cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -334,6 +354,83 @@ users:
 			},
 			expectedUsername: "none, http",
 		},
+		{
+			name: "plugin config",
+			options: &Options{
+				ConfigFile: pluginconfigFile,
+				ComponentConfig: func() kubeschedulerconfig.KubeSchedulerConfiguration {
```

Comment:
From my read of the code, the fact that these tests have both `ConfigFile` and `ComponentConfig` seems odd, since the config file is parsed as a component config and the member gets overwritten.

Anyway, I wonder if for this test, we can omit all options except `ConfigFile`. Would the test still pass? (with appropriate tweaks to the expected value. It would be nice if this test could be more focused on the features we are trying to test.

Reply:
Right!! Thanks.
I have reduced unnecessary options, more focused on the features. The ``expectedConfig`` only contains basic default.
Ref.
https://github.com/kubernetes/kubernetes/blob/b6c53beed514a439e1bfcd1a40780da545b4f87f/cmd/kube-scheduler/app/options/options_test.go#L347
https://github.com/kubernetes/kubernetes/blob/b6c53beed514a439e1bfcd1a40780da545b4f87f/pkg/scheduler/apis/config/v1alpha1/defaults.go#L42

Reply:
I think you haven't addresses @misterikkit's comment yet. We still provide both componentConfig and a config file and the config file has overlapping information with the component config. Can we move all the config from the component config to the config file and provide only the config file?